src/main/java/org/opentripplanner/middleware/bugsnag/EventSummary.java

```diff
@@ -0,0 +1,45 @@
+package org.opentripplanner.middleware.bugsnag;
+
+import org.opentripplanner.middleware.bugsnag.response.EventException;
+import org.opentripplanner.middleware.models.BugsnagEvent;
+import org.opentripplanner.middleware.models.BugsnagProject;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Event summary information provided to calling services. Response is based on static project and dynamic event
+ * information. Additional information relating to this can be found here:
+ * https://bugsnagapiv2.docs.apiary.io/#reference/organizations/collaborators/create-an-event-data-request
```

Comment:
I think this URL should be https://bugsnagapiv2.docs.apiary.io/#reference/organizations/event-data-requests/create-an-event-data-request

Reply:
Updated all reference to: https://bugsnagapiv2.docs.apiary.io/#reference/organizations/event-data-requests/create-an-event-data-request